mlrun/model.py

```diff
@@ -1581,10 +1582,13 @@ def get_templated_path(self):
         return self.full_path_template
 
     def get_absolute_path(self):
-        if self.run_id:
-            return self.full_path_template.format(run_id=self.run_id)
-        else:
-            return self.full_path_template
+        project = mlrun.get_current_project(silent=True)
+        project_name = project.name if project else None
+        path = fill_artifact_path_template(
```

Comment:
Since it's not used just for artifact-path anymore, suggest to rename this function's name to something more general, maybe `fill_path_template` or something?